test/unit/test_util.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
@tfurmston I don't think that these unit tests are executed by the CI harness. We already have tests that check if the user name is getting correctly passed around - https://github.com/Netflix/metaflow/blob/master/test/core/contexts.json#L7 and https://github.com/Netflix/metaflow/blob/master/test/core/metaflow_test/metadata_check.py#L32

Reply:
Ah, ok. Good to know. I did wonder about this point, but there were some other tests in the `unit` test folder that were structured in the same way. 

I should remove these new tests completely then?

Reply:
Yes - that will be super!

Reply:
cool. All done.